FF1Lib/BugFixes.cs

```diff
@@ -53,6 +53,7 @@ public void FixSpellBugs()
 			Put(0x33A4E, Blob.FromHex("F017EA")); // LOCK routine
 			Put(0x3029C, Blob.FromHex("0E")); // LOK2 spell effect
 			Put(0x302F9, Blob.FromHex("18")); // HEL2 effectivity
+			Put(0x3AEF3, Blob.FromHex("187D0063")); // Allows last slot in a spell level to be used outside of battle

```

Comment:
should potentially be always included instead of being with the spell fixes since it should not cause any differences to a vanilla play through